osf_tests/test_preprint_summary.py

```diff
@@ -19,11 +18,33 @@ def preprint_provider():
 def preprint(preprint_provider):
     return PreprintFactory._build(PreprintService, provider=preprint_provider)
 
+
+@pytest.fixture()
+def yesterday_at_midnight():
+    return datetime.datetime.utcnow().replace(hour=0,
+                                              minute=0,
+                                              second=0,
+                                              microsecond=0) - datetime.timedelta(days=1)
+
+
+@pytest.fixture()
+def yesterday_right_before_today():
+    return datetime.datetime.utcnow().replace(hour=23,
+                                              minute=59,
+                                              second=59) - datetime.timedelta(days=1)
+
+
+@pytest.fixture()
+def my_birthday_at_midnight():
+    return datetime.datetime(1991, 9, 25, 0)
```

Comment:
Add timezone info or you may get some lovely naive datetime issues later:
`datetime.datetime(1991, 9, 25, 0, tzinfo=pytz.utc)`